react/src/lib/components/DeckGLMap/layers/layersDefaultProps.ts

```diff
@@ -55,6 +55,7 @@ export const layersDefaultProps: Record<string, unknown> = {
         smoothShading: true,
         material: true,
         depthTest: true,
+        isZDepht: true,
```

Comment:
This mechanism for setting defaults was AFAIK added to support exposing layer settings in the toolbox. It is no longer needed and adds complexity.

We should consider applying default values according to convention. Example: https://gist.githubusercontent.com/gfox1984/d0360f921b5629dad7901e7527bd2cf2/raw/960a2d1a890503f99385442133d03d3c6aad5ecd/medium-default-props-simple-comp.ts

Reply:
This may be handled in a different pull requesst?

Reply:
Referenced in https://github.com/equinor/webviz-subsurface-components/issues/1395